src/io/csv/csv-reader.cu

```diff
@@ -422,40 +437,40 @@ gdf_error allocateGdfDataSpace(gdf_column *gdf) {
 	long num_bitmaps = (N + 31) / 8;			// 8 bytes per bitmap
 
 	//--- allocate space for the valid bitmaps
-	CUDA_TRY(cudaMallocManaged(&gdf->valid, (sizeof(gdf_valid_type) 	* num_bitmaps)));
+	CUDA_TRY(cudaMalloc(&gdf->valid, (sizeof(gdf_valid_type) 	* num_bitmaps)));
 	CUDA_TRY(cudaMemset(gdf->valid, 0, (sizeof(gdf_valid_type) 	* num_bitmaps)) );
 
 	//--- Allocate space for the data
 	switch(gdf->dtype) {
 		case gdf_dtype::GDF_INT8:
-			CUDA_TRY(cudaMallocManaged(&gdf->data, (sizeof(int8_t) * N)));
+			CUDA_TRY(cudaMalloc(&gdf->data, (sizeof(int8_t) * N)));
```

Comment:
You can make your life a little easier here with `gdf_error get_column_byte_width(gdf_column * col, int * width)` which will tell you the byte width of the column's type. That way you only need 4 casts: 1, 2, 4, 8 bytes. https://github.com/gpuopenanalytics/libgdf/blob/master/src/column.cpp#L44